src/xunit.runner.uap/Helpers.cs

```diff
@@ -15,5 +16,14 @@ internal static class Helpers
             return await folder.CreateFileAsync(test, CreationCollisionOption.ReplaceExisting);
```

Comment:
we should change this one too if possible

Reply:
I will follow up with the testResults.xml in another PR, first the output :)